docs/defects-api.yml

```diff
@@ -324,7 +295,10 @@ components:
         inspectionTypes:
           type: array
           items:
-            $ref: "#/components/schemas/inspectionType"
+            type: string
```

Comment:
Can this no still reference the type below? If not, can the `inspectionType` section be removed?

Reply:
I have no idea why this comment happened twice.